main.cpp

```diff
@@ -44,28 +44,30 @@ int main()
 struct CoffeeShop
 {
     int numBaristas = 2;
-    int numCoffeeMachines = 4;
-    int numRegisters = 2;
+    int numCoffeeMachines { 4 };
+    int numRegisters;
+    CoffeeShop() : numRegisters(2)
+    {
+        std::cout << "CoffeeShop UDT being constructed!" << std::endl;  
+    }
```

Comment:
You did this revision correctly.
However, you should have keep the implementation outside of the class and modified that constructor's implementation instead of removing it and moving the constructor implementation to inside the class
- [ ] acknowledge, but don't revise anything.

Reply:
it was sort of an experiment to see what works.